spec/rspec/rails/matchers/have_http_status_spec.rb

```diff
@@ -382,12 +382,12 @@ def create_response(opts = {})
     it_behaves_like "supports different response instances" do
       subject(:matcher) { have_redirect_status }
 
-      let(:code) { 333 }
+      let(:code) { 308 }
     end
 
     describe "matching a response" do
       it "returns true for a response with a 3xx status code" do
-        any_3xx_code = 333
+        any_3xx_code = 308
```

Comment:
Shouldn't these be dynamic depending on version? Or does 308 play ok for rails 4 etc too?